### PR TITLE
HCA zero income w/deductions

### PR DESCRIPTION
**Pull request by jbalboni**
screen shot 2017-10-17 at 4 20 32 pm
screen shot 2017-10-17 at 4 12 55 pm

### CODE REVIEW

File: src/js/hca/helpers.jsx [hunk 1]
```diff
@@ -159,3 +159,58 @@ export const disclosureWarning = (
...
+    deductibleFuneralExpenses = 0,
+    deductibleEducationExpenses = 0,
+  } = formData;
+
+  const expenses =
+    deductibleMedicalExpenses +
+    deductibleEducationExpenses +
+    deductibleFuneralExpenses;
+
+  return income > expenses;
```
Comment (jbalboni) [inline src/js/hca/helpers.jsx]: Warning message copy doesn't match this logic right now.

File: src/js/common/components/form-elements/ExpandingGroup.jsx [hunk 2]
```diff
@@ -36,5 +36,6 @@ export default function ExpandingGroup({ children, open, showPlus, additionalCla
 ExpandingGroup.propTypes = {
   open: PropTypes.bool,
   additionalClass: PropTypes.string,
-  showPlus: PropTypes.bool
+  showPlus: PropTypes.bool,
+  expandedContentId: PropTypes.string
```
Comment (cehsu) [inline src/js/common/components/form-elements/ExpandingGroup.jsx]: Should some of these be made required?

**Review by cehsu — APPROVED** [review]
LGTM. Had one minor question. Excited about the new tooltip component!

File: src/js/common/components/form-elements/ExpandingGroup.jsx (hunk 2 above)
Comment (jbalboni) [reply src/js/common/components/form-elements/ExpandingGroup.jsx]: Just open, I think.

**Pull request merged**